### PR TITLE
Joshua g/popper

### PR DESCRIPTION
**Pull request by joshuagarrido**

### CODE REVIEW

File: frontend/src/components/CRUD/Users.tsx
Comment (nickogoodwin) [inline frontend/src/components/CRUD/Users.tsx:1]: haha it looks like Prettier did some formatting for you.

File: frontend/src/pages/Profile/Admin/admin.tsx [hunk 1]
```diff
@@ -229,7 +229,46 @@ const Column = ({
             /></View>
       </View> */
 }
+//User Crude
+const createUser = (userData: User) => {
```
Comment (nickogoodwin) [inline frontend/src/pages/Profile/Admin/admin.tsx:233]: this method needs to be declared within the UsersTableHeader component, not outside of it

File: frontend/src/pages/Profile/Admin/admin.tsx [hunk 2]
```diff
@@ -229,7 +229,46 @@ const Column = ({
...
+    })
+    .catch((error) => {
+      // handle error
+      console.log(error.response.data);
+    });
+};
+ */}
 
+ 
+const deleteUser = (userId:string) => {
```
Comment (nickogoodwin) [inline frontend/src/pages/Profile/Admin/admin.tsx:261]: this method needs to be inside of the `UserElement `component

File: frontend/src/pages/Profile/Admin/admin.tsx [hunk 3]
```diff
@@ -253,19 +292,49 @@ const UsersTableHeader = ({
             <TextInput style={styles.createNew} placeholder={label} />
           </View>
         ))}
+        <TouchableOpacity onPress={() => createUser({ name: '', email: '', role: '', phone: '', address: { street: '', city: '', state: '', zip: '', country: '' } })}>
```
Comment (nickogoodwin) [inline frontend/src/pages/Profile/Admin/admin.tsx:295]: hmmmm :thinking_face: ,
These really should be `useState` variables that are updated via the `value` of their respective `<TextInput>`'s. However, the TextInput's are being mapped via the `labels`, @william-beukelman We need
`<TextInput value={name} />`, how could we accomplish this while preserving your label mapping?

**Review by nickogoodwin — CHANGES_REQUESTED** [review]
needs some changes

File: frontend/src/pages/Profile/Admin/admin.tsx (hunk 3 above)
Comment (williambeukelman) [reply frontend/src/pages/Profile/Admin/admin.tsx:295]: image

**Pull request closed without merging**